python/python/glide/async_commands/server_modules/ft_options/ft_aggregate_options.py

```diff
@@ -0,0 +1,322 @@
+# Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Mapping, Optional
+
+from glide.async_commands.server_modules.ft_options.ft_constants import (
+    FtAggregateKeywords,
+)
+from glide.constants import TEncodable
+
+
+class FtAggregateClause(ABC):
+    """
+    Abstract base class for the FT.AGGREGATE command clauses.
+    """
+
+    def toArgs(self) -> List[TEncodable]:
```

Comment:
Python code uses sneak_case - please update throughout
```suggestion
    @abstractmethod
    def to_args(self) -> List[TEncodable]:
```

Reply:
Updated for the entire file.